foodsaving/pickups/models.py

```diff
@@ -288,7 +288,7 @@ class PickupDateCollector(BaseModel):
     class Meta:
         db_table = 'pickups_pickupdate_collectors'
         unique_together = (('pickupdate', 'user'), )
-        ordering = ['-created_at']
+        ordering = ['created_at']
```

Comment:
No migration needed?

Reply:
Ah right. It does create a migration, so I better commit it. But it doesn't have any effect on the database, so it works without.